calc.js

```diff
@@ -1,3 +1,124 @@
 // Build a simple webpage that has an interactive calculator using JavaScript
 
-// You will want some kind of html element to display the calculations (maybe a span) and a few buttons to press to input numbers and operators
\ No newline at end of file
+// You will want some kind of html element to display the calculations (maybe a span) and a few buttons to press to input numbers and operators
+
+let input = document.getElementById("Input");
+let history = "";
+const divide = document.getElementById("Divide");
```

Comment:
you can simplify this by using `document.querySelectorAll('.someClassGoesHereThatTheyAllShare')`
that will give you an array of html elements and you won't need variables for each individual button